src/pages/components/table.mdx

```diff
@@ -239,6 +285,9 @@ import TableVirtualizedCode from '!!raw-loader!../../examples/table/TableVirtual
 
 <Component components={props.data.mdx.components} componentName="overflowButton" />
 
+The OverflowButton component enables implementation of an overflow menu in a [Cell](#cell). Provide an
+`aria-label` attribute or use another technique to ensure it is accessible. See [example](#overflow) for details.
```

Comment:
I believe our `aria-label` guidance here is outdated. Garden currently prefers the use of a `Tooltip` which satisfies accessibility needs for both mouse and keyboard users. The example itself and documented guidance should probably be updated to reflect this.

Compare with existing toolbar icons shown in the footer of each code example on the component pages. This is what we're looking for when an icon is the only visual information source on the page.

Reply:
Sorry for the delay, @jzempel . I updated the guidance as well as the example to use `Tooltip` instead. Let me know if this is not what you had in mind.